lib/internal/event_target.js

```diff
@@ -609,16 +617,16 @@ class EventTarget {
       // not prevent the event target from GC.
```

Comment:
@benjamingr Is this comment still relevant? seems to me like this was handled already